docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc

```diff
@@ -17,15 +16,64 @@ field for autocompletion.
 [role="screenshot"]
 image::images/agent-tags.png[Agents list filtered to show agents with the staging tag]
 
-If you haven't applied tags to any {agent}s yet, the list will be empty.
+If you haven't added tags to any {agent}s yet, the list will be empty.
+
+[discrete]
+[[add-tags-in-fleet]]
+== Add, remove, or rename tags in {fleet}
+
+You can use {fleet} to add, remove, or rename tags applied to one or more
+{agent}s.
+
+Want to add tags when enrolling from a host instead? See
+<<add-tags-at-enrollment>>.
+
+To manage tags in {fleet}:
+
+. On the **Agents** tab, select one or more agents.
+
+. From the **Actions** menu, click **Add / remove tags**.
++
+[role="screenshot"]
+image::images/add-remove-tags.png[Screenshot of add / remove tags menu]
++
+TIP: Make sure you use the correct **Actions** menu. To manage tags for a single
+agent, click the ellipses button under the **Actions** column. To manage tags
+for multiple agents, click the **Actions** button to open the bulk actions menu.
+
+. In the tags menu, perform an action:
++
+[options,header]
+|===
+|To... | Do this...
+
+|Create a new tag
+|Type the tag name and click **Create new tag...**. Notice the tag name has
+a check mark to show that the tag has been added to the selected agents.
+
+|Rename a tag
+|Hover over the tag name and click the ellipses button. Enter a new name.
```

Comment:
we could be more explicit here, something like `Enter a new name, press Enter or click outside.`